linked_list/main.py

```diff
@@ -62,34 +68,166 @@ class LinkedList(AbstractLinkedList):
     """
 
     def __init__(self, elements=None):
-        pass
+        self.elements = []      
+        if (elements == None or len(elements) <1):
+            #self.start, self.end = Node(None, None), Node(None, None)
+            self.start, self.end = None, None
+        if elements != None and len(elements) > 0:
+            self.elements = elements
+            index = 1
+            for e in range(len(self.elements)):
+                try:
+                    e = Node(e, Node(self.elements[index]))
+                except IndexError:
+                    e = Node(e)
+            try:
+                self.start = Node(self.elements[0], self.elements[1])
```

Comment:
Careful that you are assigning Node.next to point to an element, when you want it to point to a Node object